tests/cbmc/stubs/s2n_evp_signing_supported.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cbmc_proof/nondet.h>
+
+static bool is_set = false;
+static bool is_evp_signing_supported = false;
+
+bool s2n_evp_signing_supported()
```

Comment:
This follows the model set by [s2n_is_in_fips_mode](https://github.com/aws/s2n-tls/blob/main/tests/cbmc/stubs/s2n_is_in_fips_mode.c) and [s2n_libcrypto_is_awslc](https://github.com/aws/s2n-tls/blob/main/tests/cbmc/stubs/s2n_libcrypto_is_awslc.c)